en/mkdocs.yml

```diff
@@ -29,62 +29,83 @@ edit_uri: https://github.com/wso2/docs-is/edit/master/en/docs/
 copyright: WSO2 Identity Server - Documentation
 
 # Configuration
-base_url: /
 theme: material
 theme:
   name: material
   palette:
-    primary: 'deep-orange'
-    accent: 'deep-orange'
-  custom_dir: 'theme/material'
-  logo: 'images/logo.svg'
+    primary: deep-orange
+    accent: deep-orange
+  custom_dir: theme/material
+  logo: images/logo.svg
   highlightjs: true
   feature:
     tabs: true
   language: 'en'
+  highlightjs: true
+  hljs_languages:
+    - toml
+  hljs_style: github
   
 # Navigation
 nav:
-- Key Concepts:
-      - 'Single Sign-On and Identity Federation': 'key-concepts/single-sign-on-and-identity-federation_.md'
-      - 'Identity Provisioning and its Standards': 'key-concepts/identity-provisioning-and-its-standards.md'
-      - 'Access Control and Entitlement Management': 'key-concepts/access-control-and-entitlement-management.md'
-      - 'Identity Anti-patterns and the Identity Bus': 'key-concepts/identity-anti-patterns-and-the-identity-bus.md'
-- Basic Tutorials:
-      - 'Adaptive Authentication': 
-        - 'Adaptive Authentication Overview': 'tutorials/adaptive-authentication-overview.md'
-        - 'Configuring a Service Provider for Adaptive Authentication': 'tutorials/configuring-a-service-provider-for-adaptive-authentication.md'
-        - Adaptive Authentication Scenarios:
-          - 'Configuring Role-Based Adaptive Authentication': 'tutorials/configuring-role-based-adaptive-authentication.md'
-          - 'Configuring User-Age-Based Adaptive Authentication': 'tutorials/configuring-user-age-based-adaptive-authentication.md'
-          - 'Configuring Tenant-Based Adaptive Authentication': 'tutorials/configuring-tenant-based-adaptive-authentication.md'
-- References:
-      - 'Configuration Catalog': 'references/config-catalog.md'
-      - 'Default Ports of WSO2 Products': 'references/default-ports-of-wso2-products.md'
-      - 'API Permissions': 'references/api-permissions.md'
-      - 'Adaptive Authentication JS API Reference': 'references/adaptive-authentication-js-api-reference.md'
+  - 'Key Concepts':
```

Comment:
This PR is missing the nav fixes made in this PR: https://github.com/wso2/docs-is/pull/19/

Reply:
Thanks! fixed it.